backupdrupal.sh

```diff
@@ -0,0 +1,49 @@
+# kullanım
+# Drush 9'da 'drush ard' komutunun kaldırılması nedeni ile
+# içinde drush sql-dump komutunun da bulunduğu backup kodu
+# scriptin parametrelerinde her ikisinin de çift tırnak
+# içinde girilmiş olması gerekir.
+# ikinci parametrenin sonundaki / işareti olmalıdır.
+# sonuç olarak kullanıcı dizininizin içinde
+# içinde veritabanı yedeğinin ve tüm dosyaların olduğu 
+# uzantısı tar.gz olan tek bir arşiv dosyası oluşacak
+# tıpkı drush 8'deki 'drush ard' komutunda elde edildiği gibi
+# bash backupdrupal.sh "drupal_site_adi" "/drupal/kurulumu/kok/dizini/"
+
+# $1 değişkeni betiğe gönderdiğiniz "drupal_site_adi"
+# $2 değişkeni betiğe gönderdiğiniz "/drupal/kurulumu/kok/dizini/"
+# değerini alır.
+
+# Drush Launcher Version: 0.6.0
+# Drush Commandline Tool 9.4.0
+# ile test edilmiştir.
+
+# backup zamanı oluşturulur
+timestamp=$(date +%Y%m%d_%H%M%S)
+
+# backup dizini tanımlanır.
+backupdirbase=$HOME/backups
+backupdir=$HOME/backups/$timestamp
+
+# backup dizini oluşturulu
```

Comment:
# backup dizini oluşturulu   =>   # backup dizini oluşturulur

Reply:
ne kadar kaba duruyor bu yaptığım yorum

Reply:
bunu şu şekilde yaparsak daha iyi olur.
1. PR'i reddet
2. PR'i reddederken hatalı satıra referans ver.

Reply:
Yani reddetmek gerekir bu durumda...

Reply:
Böyle bir şey için reddetmek saçma olmaz mı :)

Reply:
Ok. Then please merge it and then file an ıssue for the typo.